src/components/user-preview/user-preview.css

```diff
@@ -1,3 +1,45 @@
+.UserPreview,
```

Comment:
To prevent other components to use this CSS rules, all the rules should be prefixed with .UserPreview in the selector.

Try to render the component in small screens (for example 300px). It is not responsive. Consider using grid to solve the problem. Check the implementation of StatusButton (github) to see how it is implemented there using grid

I would simplify the inner/outer circle by creating only 1 div desired size, border radius and use the border as outer circle and the background as inner circle. 

max-width of component will be 100%.